### PR TITLE
UI cleanup round2

### PR DESCRIPTION
**Pull request by karenkun**
## Description

Refactoring some of the styles/components to work better with Chakra.
- Filled in content for Contact us screen
- Filled in content for Page not found screen
- Moved more css into the react component where possible
- Heading component now has variant "yellowline" for when you want to have the decorative yellow line appear
- Went through the app to make sure all the h1, h2, h3 are appear in the correct nesting and page order throughout app
  -  Standardized their font sizes when they appear on pages (within <Container>) so we don't have to manually set it each time
- Grid tables to use smaller text; its paginator to show first 3 pages
- Improved the theme's Button, Link, ManageMenuItem (in dropdown menus)
- Update blue header's graphic to be a housing one

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📦 Chore (Release)
- [ ] ✅ Test

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-671
https://hous-bssb.atlassian.net/browse/BPHH-797
https://hous-bssb.atlassian.net/browse/BPHH-480
https://hous-bssb.atlassian.net/browse/BPHH-858
https://hous-bssb.atlassian.net/browse/BPHH-793

## UI Accessibility Checklist

- Navigate around app to see the order of headings h1, h2 , h3 appear in the right order not skipping
- Links have `isExternal` where applicable

_Note the following checklist is not an exhaustive list. Check
the [WebAIM checklist](https://webaim.org/standards/wcag/checklist) for
a user-friendly checklist with more details. Aim to meet at least *AA* conformance level where applicable. Check
the [WCAG guidelines](https://webaim.org/standards/wcag/checklist) for the full guidelines._

- [ ] Markup uses semantic HTML?
- [ ] Additional context added through `aria-roles` and `aria-labels`?
- [ ] Checked
      with [Wave Browser Extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
      for accessibility errors?
- [ ] Usable with a keyboard and navigable in a logical order?
- [ ] Usable using a screen reader (e.g. Voiceover for Mac) with enough context for the user?

## General Checklist

- [ ] ✅ Provide tests for your changes where relevant
- [ ] 📝 Use descriptive commit messages
- [ ] 📗 Update any related documentation and include any relevant screenshots
- [ ] 📱 For UI-related tasks, ensure responsive design is implemented across multiple screen size
- [ ] 🗂️ Move your relevant story/task to the _Ready for Code Review_ state

### CODE REVIEW

File: app/frontend/components/domains/jurisdictions/jurisdiction-screen.tsx [hunk 1]
```diff
@@ -72,7 +72,7 @@ export const JurisdictionScreen = observer(() => {
 
   return (
     <Flex as="main" direction="column" w="full" bg="greys.white">
-      <BlueTitleBar title={qualifiedName} imageSrc={"/images/jurisdiction-bus.svg"} />
+      <BlueTitleBar title={qualifiedName} />
```
Comment (bkmaibach) [inline app/frontend/components/domains/jurisdictions/jurisdiction-screen.tsx:75]: no more bus image? :'[

File: app/frontend/components/domains/misc/contact-screen.tsx [hunk 2]
```diff
@@ -1,6 +1,73 @@
...
+        <strong>{t("site.contactInstructions_1")}</strong>
+      </Text>
+
+      <Heading as="h2" variant="yellowline" mt="16">
+        {t("site.contactInstructions_2")}
+      </Heading>
+      <Text>
+        {t("site.contactInstructions_3")}
+        {/*
+					<Link ml="1" href="">
```
Comment (bkmaibach) [inline app/frontend/components/domains/misc/contact-screen.tsx:22]: crufty cruft

**Review by bkmaibach — CHANGES_REQUESTED** [review]
looks good

File: app/frontend/components/domains/jurisdictions/jurisdiction-screen.tsx (hunk 1 above)
Comment (karenkun) [reply app/frontend/components/domains/jurisdictions/jurisdiction-screen.tsx:75]: Its gonna be a house now, because housing

**Pull request merged**